src/core/issues-notes.js

```diff
@@ -141,31 +141,32 @@ export function run(conf, doc, cb) {
     issueBase = conf.issueBase;
   if ($ins.length) {
     if (conf.githubAPI) {
-      ghFetch(conf.githubAPI)
-        .then(function(json) {
+      (async function(githubAPI) {
+        try {
+          let json = await ghFetch(githubAPI);
           issueBase = issueBase || json.html_url + "/issues/";
-          return fetchIndex(json.issues_url, {
+          let issues = fetchIndex(json.issues_url, {
             // Get back HTML content instead of markdown
             // See: https://developer.github.com/v3/media/
-            Accept: "application/vnd.github.v3.html+json",    
+            Accept: "application/vnd.github.v3.html+json",
           });
-        })
-        .then(function(issues) {
+          
           issues.forEach(function(issue) {
```

Comment:
This can execute before ```issues``` has actually got its value ```fetchIndex```. You need to add another waiting step here.

Reply:
hm, you are saying the waiting step should go right before `fetchIndex`, like @OrionStar25 mentioned?

I will make the changes according to your feedback and push again soon

Reply:
Yes, precisely.